### PR TITLE
Change Kubebuilder RBAC Annotation Groups

### PR DESCRIPTION
**Pull request by Mr-Linus**

### CODE REVIEW

**Review by jasonliu747 — COMMENTED** [review]
Nice catch, thanks for your contribution!
Btw, please follow the instruction to fix DCO error.

**Comment by Mr-Linus** [comment]
> Nice catch, thanks for your contribution! Btw, please follow the instruction to fix DCO error.

Fixed it.

**Comment by jasonliu747** [comment]
@FillZpp CI keeps failing, PTAL.

**Comment by Mr-Linus** [comment]
> @FillZpp CI keeps failing, PTAL.

@jasonliu747 It seems that the ci version causes the failing,because the  [Merged PR#41](https://github.com/koordinator-sh/koordinator/pull/41) still has the problem.I try to update the ci version to 1.45 and fixed it.

**Comment by jasonliu747** [comment]
> 
> @jasonliu747 It seems that the ci version causes the failing,because the [Merged PR#41](https://github.com/koordinator-sh/koordinator/pull/41) still has the problem.I try to update the ci version to 1.45 and fixed it.

That’s weird, it worked fine last week. Thanks for the fix though.
/lgtm

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ on:
 env:
   # Common versions
   GO_VERSION: '1.17'
-  GOLANGCI_VERSION: 'v1.42'
+  GOLANGCI_VERSION: 'v1.45.2'
```
Comment (jasonliu747) [inline .github/workflows/ci.yaml:15]: I updated to the latest version and skip cache in #43 , that would fix the ci issue.

**Pull request merged**